firmware/common/portapack_persistent_memory.cpp

```diff
@@ -337,7 +176,7 @@ struct data_t {
     int32_t modem_baudrate;
     int32_t modem_repeat;
 
-    // Play dead unlock
+    // Play dead unlock (Used?)
```

Comment:
Play dead was disabled at some points because it was not working correctly / using bin space.
It was intended to be like the 'lock' feature of a flipper.
We should poll but I'm afraid they will ask us to bring it back X-D

Reply:
That's a lot of space we could have back for settings if we don't :)